src/software/thunderscope/gl/graphics/gl_rect.py

```diff
@@ -1,23 +1,35 @@
-from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.Qt import QtGui
 from pyqtgraph.opengl import *
+from pyqtgraph.opengl.GLGraphicsItem import GLGraphicsItem
 
+import numpy as np
 
-class GLRect(GLGridItem):
+
+class GLRect(GLLinePlotItem):
     """Displays a rectangle parallel to the x-y plane"""
 
-    def __init__(self, color=(255, 255, 255, 127.5)):
+    def __init__(
+        self,
+        parentItem: GLGraphicsItem = None,
+        color: QtGui.QColor = (255, 255, 255, 127.5),
+        line_width: float = 1.0,
```

Comment:
Not sure if you've changed anything, or if this always worked on my PC, but the increased `line_width` seems great on my PC. Is it possible to make `3.0` the default for all of the main shapes? Including the robot outlines, path, robot/ball velocity, centre circle (if you don't think it looks worst). Since those still look quite thin on my monitor.

Reply:
I just made the color less transparent which is why it looks more visible. I still don't think the line width param does anything (unless you tried changing it and it worked?) Regardless I can set the default width to 3

Reply:
I can verify that I did change it to something like 30 and it did work. And 1 and 3 do look different on my computer (native ubuntu).

Reply:
OK, that's great